spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+  describe "GET start page" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
```

Comment:
I think it's better to use helpers
[`expect(response).to have_http_status :ok`](http://www.relishapp.com/rspec/rspec-rails/v/3-2/docs/matchers/have-http-status-matcher#using-in-a-controller-spec)